### DISCUSSION TITLE
MenuBar with event instead of window

### DISCUSSION THREAD

**Discussion by martinpl** [NativePHP/laravel#246] [Ideas]
So I want to toggle the main window when clicking on the tray icon. Currently this is possible only with a workaround that is based on MenuBar window events. It would be nice if we could do the same thing as in Menu class, so just passing the event without creating a window.

Ex.
```
MenuBar::create()
    ->event(\App\Events\MyEvent::class) <-- proposal
    ->withContextMenu(
        Menu::new()
            ->event(\App\Events\MyEvent::class, 'Trigger my event') <-- exists
            ->separator()
            ->link('https://nativephp.com', 'Learn more…')
            ->separator()
            ->quit()
    )
    ->onlyShowContextMenu();
```

**Comment by simonhamp** [comment]
Added in [nativephp/laravel v0.6.3](https://github.com/NativePHP/laravel/releases/tag/0.6.3) and [nativephp/electron v0.8.6](https://github.com/NativePHP/electron/releases/tag/0.8.6).

Docs incoming.
